src/content/language/concurrency.md

```diff
@@ -253,9 +253,7 @@ together. For more information on this concurrency model, read about
 the [Actor model](https://en.wikipedia.org/wiki/Actor_model).
 
 Using isolates, your Dart code can perform multiple independent tasks at once,
-using additional processor cores if they're available. Isolates are like threads
-or processes, but each isolate has its own memory and a single thread running an
-event loop. 
+using additional processor cores if they're available.
```

Comment:
This is the only part that's missing from the paragraph earlier on. Could you delete this line here (so the entire paragraph is deleted from 255-258) but add this clause (`using additional processor cores if they're available.`) between lines 241 and 242?

So delete:
```suggestion
```

And then, above, the full paragraph should look like:
```md
Instead of threads, all Dart code runs inside isolates.
Using isolates, your Dart code can perform multiple independent tasks at once,
using additional processor cores if they're available.
Isolates are like threads or processes, but each isolate has its own memory
and a single thread running an event loop.
```